ci/pipeline

```diff
@@ -221,3 +235,34 @@ def jenkins(): Unit = {
   if(utils.isPullRequest) pr()
   else master()
 }
+
+/**
+ * Executes the Community Release which includes:
+ *  1. tarball with version details
+ *  2. release of docker with that version
+ *  3. git tag of version (pushed to server)
+ *
+ * @return Version and artifact description of Marathon build.
+ */
+@main
+def release(releaseVersion: String, gitSha: String, latest: String = "False"): Unit = {
+  println(s"Releasing version: $releaseVersion ")
+  %('git, "checkout", gitSha)
+  %('git, "tag", "-a", releaseVersion, "-m", releaseVersion)
+  val version = build()
```

Comment:
🤔 hm, it feels odd that we pass a release version and then have something returned by `build()`. I know the background but am wondering if we could avoid this. Also `version` is not used in this method.